test/postgresql/import_test.rb

```diff
@@ -1,4 +1,7 @@
 require File.expand_path(File.dirname(__FILE__) + '/../test_helper')
 require File.expand_path(File.dirname(__FILE__) + '/../support/postgresql/import_examples')
 
+pg_version = ENV["PG_VERSION"] || "0"
```

Comment:
@zdennis I refactored this so that ENV variable PG_VERSION is no longer required. I didn't realize I could grab PostgreSQL version from `ActiveRecord::ConnectionAdapters::PostgreSQLAdapter#postresql_version`.